CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to the Lethe project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
+### [Master] - 2025-08-08
+
+### Fixed 
+
+- MINOR The mortar update was not accounting for the updated constraints when rotating the mesh, and the initial condition was being erased when updating the mortar information. This has been fixed. This PR also changes the way of reading msh files, so that two separate rotor/stator meshes are taken as input (the same way it was being done for dealii grids). [#1614](https://github.com/chaos-polymtl/lethe/pull/1614)
```

Comment:
```suggestion
- MINOR The mortar update was not accounting for the updated constraints when rotating the mesh, and the initial condition was being erased when updating the mortar information. This has been fixed. This PR also changes the way msh files are read, so that two separate rotor/stator meshes are taken as input (the same way it was done for dealii grids). [#1614](https://github.com/chaos-polymtl/lethe/pull/1614)
```